docs/core_docs/docs/integrations/document_loaders/web_loaders/dropbox.mdx

```diff
@@ -0,0 +1,138 @@
+---
+hide_table_of_contents: true
+sidebar_class_name: node-only
+---
+
+# Dropbox Loader
+
+The `DropboxLoader` allows you to load documents from Dropbox into your LangChain applications. It retrieves files or directories from your Dropbox account and converts them into documents ready for processing.
+
+## Overview
+
+Dropbox is a file hosting service that brings all your files—traditional documents, cloud content, and web shortcuts—together in one place. With the `DropboxLoader`, you can seamlessly integrate Dropbox file retrieval into your projects.
+
+## Setup
+
+1. Create a dropbox app, using the [Dropbox App Console](https://www.dropbox.com/developers/apps/create).
+2. Ensure the app has the `files.metadata.read`, `files.content.read` scope permissions:
+3. Generate the access token from the Dropbox App Console.
+4. To use this loader, you'll need to have Unstructured already set up and ready to use at an available URL endpoint. It can also be configured to run locally.
+   See the docs [here](https://www.dropbox.com/developers/apps/create) for information on how to do that.
+5. Install the necessary packages:
+
+   ```bash npm2yarn
+   npm install @langchain/community @langchain/core dropbox
+   ```
+
+## Usage
+
+### Loading Specific Files
+
+To load specific files from Dropbox, specify the file paths:
+
+```typescript
+import { DropboxLoader } from "@langchain/community/document_loaders/web/dropbox";
+
+const loader = new DropboxLoader({
+  clientOptions: {
+    accessToken: "your-dropbox-access-token",
+  },
+  unstructuredOptions: {
+    apiUrl: "http://localhost:8000/general/v0/general", // Replace with your Unstructured API URL
```

Comment:
Let's emphasize somewhere that this wraps Unstructured

Should we call this `DropboxUnstructuredLoader` instead?